src/napari_omero/widgets/tree_model.py

```diff
@@ -34,7 +35,9 @@ def hasChildren(self):
         return bool(self.wrapper.CHILD_WRAPPER_CLASS)
 
     def numChildren(self) -> int:
-        return self.wrapper.countChildren()
+        if self._childCount is None:
+            self._childCount = self.wrapper.countChildren()
+        return self._childCount
```

Comment:
to fix the mypy error, you can either this above:
```
self._childCount: Optional[int] = None
```
or delete `self._childCount = None` and change this to 
```
        if not hasattr(self, '_childCount'):
            self._childCount: int = self.wrapper.countChildren()
        return self._childCount
```

Reply:
working on this right now, adding lazy loading.  much better.  thanks for the tip on getObjects

Reply:
I'm gonna push the merged changes ... so pull if you're also working on it

Reply:
nevermind... looks like I can't push to your branch.  I'll just merge this one